### PR TITLE
Fix: polishing Single Hotel Description

### PR DESCRIPTION
**Pull request by RobinCsl**

### CODE REVIEW

File: app/hotels/src/singleHotel/hotelInformation/description/Description.js [hunk 1]
```diff
@@ -24,12 +25,7 @@ type Props = {
 
 export const Description = (props: Props) => (
   <View style={styles.container}>
-    <Text style={styles.title}>
-      <Translation
-        id="single_hotel.description.description"
-        textTransform="uppercase"
-      />
-    </Text>
+    <DescriptionTitle translationId="single_hotel.description.description" />
```
Comment (tbergq) [inline app/hotels/src/singleHotel/hotelInformation/description/Description.js]: could we rather send the translation itself as prop, as we normally do

File: app/hotels/src/singleHotel/hotelInformation/description/Description.js [hunk 2]
```diff
@@ -39,7 +35,10 @@ export const Description = (props: Props) => (
         <Translation passThrough={idx(props, _ => _.hotel.summary)} />
       </Text>
     </ReadMore>
-    <Facilities facilities={idx(props, _ => _.hotel.facilities)} />
+    <View style={styles.facilitiesContainer}>
+      <DescriptionTitle translationId="single_hotel.description.equipment" />
```
Comment (tbergq) [inline app/hotels/src/singleHotel/hotelInformation/description/Description.js]: here also

File: app/hotels/src/singleHotel/hotelInformation/description/DescriptionTitle.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+type Props = {|
+  +translationId: TranslationKeysType,
+|};
+
+export default function DescriptionTitle(props: Props) {
+  return (
+    <Text style={styles.title}>
+      <Translation id={props.translationId} textTransform="uppercase" />
+    </Text>
```
Comment (tbergq) [inline app/hotels/src/singleHotel/hotelInformation/description/DescriptionTitle.js]:
```
<Text>
{props.title}
</Text>
```

**Comment by RobinCsl** [comment]
@tbergq  🚀

**Pull request merged**